scripts/west_commands/blobs.py

```diff
@@ -5,12 +5,14 @@
 import argparse
 import os
 import re
+import shutil
 import sys
 import textwrap
 from pathlib import Path
 from urllib.parse import urlparse
 
 from west.commands import WestCommand
+from west.configuration import config
```

Comment:
```suggestion
from west.configuration import wconfig
```

`config` is incredibly vague. It is a variable, a function, a module, an object?